2_pvc_destination_gen/vars/storage-class-mappings.yml.example

```diff
@@ -1,4 +1,4 @@
 mig_storage_class_mappings:
   glusterfs-storage_RWO: ocs-storagecluster-ceph-rbd
   glusterfs-storage_RWX: ocs-storagecluster-cephfs
-  glusterfs-storage-block_RWO: ocs_storagecluster-ceph-rbd
\ No newline at end of file
+  glusterfs-storage-block_RWO: ocs-storagecluster-ceph-rbd
```

Comment:
nice catch

Reply:
the script caught it, kept complaining that SC does not exist :P

Reply:
i wondered :)